src/main/java/core/basesyntax/RandomColor.java

```diff
@@ -1,10 +1,18 @@
 package core.basesyntax;
 
+import java.util.function.Supplier;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для случайного
  * выбора цвета</p>
  */
-public class RandomColor {
+public class RandomColor implements Supplier<String> {
     private static final String[] COLORS = {"red", "green", "blue", "yellow", "pink",
             "black", "white"};
+
+    @Override
+    public String get() {
+        int i = (int) (Math.random() * (COLORS.length - 1));
```

Comment:
1) will we be able to get 6?
2) let's remove this variable

Reply:
1. I was wrong - 1 is unreachable for Math.random. Fixed.
2. Done.